### PR TITLE
Add standalone `clightning-rest` service, add lndconnect-onion for clightning

### PR DESCRIPTION
**Pull request by erikarvstedt**
#### Main commit: `add standalone clightning-rest service`

- Rename `services.rtl.cl-rest` to `services.clightning-rest`.
  `clightning-rest` is generally useful for connecting external REST clients to clightning.

- Add a dedicated network namespace in netns-isolation.

- Add nodeinfo entry.

- Add datadir (which contains REST auth data) to backups.

I've tested the datadir migration with [this script](https://gist.github.com/59a5ad58021eaf47a8b3acfeb94cfa62) (and by deploying it in production).

#### Add lndconnect and Zeus onion support for clightning
See commits `lnd-rest-onion-service.nix: ...`, `docs/services: update Zeus usage section, add clightning`.

### CODE REVIEW

File: docs/services.md [hunk 1]
```diff
@@ -44,6 +44,61 @@ You can find the `<onion-address>` with command `nodeinfo`.
...
+   - Add a new node
+   - Select `Scan lndconnect config` (at the bottom) and scan the QR code
+   - For clightning: Set `Node interface` to `c-lightning-REST`
+   - Click `Save node config`
+   - Start sending sats privately
+
+### Additional lndconnect features
+Create plain text URLs or QR code images:
+```
+lndconnect-onion --url
```
[the remark this answers is not in the document]
Comment (erikarvstedt) [reply docs/services.md:94]: It's useful for testing and debugging, and it directly shows what data is exposed.

Zeus indeed fails when the QR-encoded URL contains a trailing newline.
The following works:
```bash
lndconnect-onion --url | tr -d "\n" | nix run nixpkgs#qrencode -- -o qr.png
```

[the remark this answers is not in the document]
Comment (jonasnick) [reply docs/services.md:94]: Thanks. Got it to work. The problem was that qemu was truncating the output.

**Review by jonasnick — COMMENTED** [review]
ACK. Tested with and without netns. Needs trivial rebase.

**Review by jonasnick — APPROVED** [review]
ACK 8e1ad6e3a85a8b534b0a03fd7760cf62cde0f205

**Pull request merged**